tests/settings/FunctionMetadataTest.py

```diff
@@ -98,8 +98,9 @@ def check_function_metadata_data_set(self,
                                          retval_desc="",
                                          non_optional_params=None,
                                          optional_params=None):
-        non_optional_params = non_optional_params or {}
-        optional_params = optional_params or {}
+
```

Comment:
@Makman2 do we really want this change in cases like this ?
if it's falsey it's enough for us to use the alternative right ?